enigma-cli/src/main/java/cuchaz/enigma/command/checks/CheckFailureException.java

```diff
@@ -0,0 +1,7 @@
+package cuchaz.enigma.command.checks;
+
+public class CheckFailureException extends RuntimeException {
```

Comment:
I personally don't recommend overriding this. I see no point of this being an exception.

In fact, if the throwable/exception qualities are desirable, I suggest overriding `fillInStackTrace` to be no-op; otherwise the performance cost of creating exceptions are intensive, especially in loops